pom.xml

```diff
@@ -20,6 +21,9 @@
     <developers>
         <developer/>
     </developers>
+    <modules>
+        <module>app</module>
```

Comment:
ja bym raczej zrobil ze to co nazwalas app czyli jak mniemam root niech to sie nazywa trainsmart a modul glowny niech sie nazywa application